apps/src/StudioApp.js

```diff
@@ -757,6 +767,82 @@ StudioApp.prototype.onResize = function() {
   this.resizeToolboxHeader();
 };
 
+StudioApp.prototype.onMouseDownVizResizeBar = function (event) {
+  // When we see a mouse down in the resize bar, start tracking mouse moves:
+
+  this.onMouseMoveBoundHandler = _.bind(this.onMouseMoveVizResizeBar, this);
+  document.body.addEventListener('mousemove', this.onMouseMoveBoundHandler);
+
+  event.preventDefault();
+};
+
+function applyTransformScaleToChildren(element, scale) {
+  for (var i = 0; i < element.children.length; i++) {
+    element.children[i].style.transform = scale;
```

Comment:
Might be worth noting in comments somewhere that the styles set in adjustAppSizeStyles are being overridden here.